carmen.gemspec

```diff
@@ -5,7 +5,7 @@
 
 Gem::Specification.new do |s|
   s.name = %q{carmen}
-  s.version = "0.2.15"
```

Comment:
In the future I don't think we should be bumping versions on forked gems. Just keep that in mind.